Dockerfile

```
@@ -1,3 +1,7 @@
-FROM scratch
-COPY gcp-artifact-registry-docker-proxy /
-ENTRYPOINT ["/gcp-artifact-registry-docker-proxy"]
+FROM alpine
```

Recommend pinning this to a specific version of alpine. `latest` [refers to 3.19.1](https://hub.docker.com/layers/library/alpine/3.19.1/images/sha256-6457d53fb065d6f250e1504b9bc42d5b6c65941d57532c072d929dd0628977d0?context=explore)

```suggestion
FROM alpine:3.19.1
```